tools/client-plugins/browser-scripts/index.d.ts

```diff
@@ -16,13 +16,20 @@ export interface PythonDocument extends FrameDocument {
   __runPython: (code: string) => Promise<PyodideInterface>;
 }
 
+export interface Source {
+  index: string;
+  contents?: string;
+  editableContents: string;
+  original: { [key: string]: string };
+}
```

Comment:
I don't like repeating the Source type, but Idk, how we do handle this.

It isn't used in tools, but it's part of the type used in the tools, `string` isn't the actual type of the argument.

- should I revert it and use `as` again? or treat `Source` as part `InitTestFrameArg` and keep it in tools and import it to client?